ouroboros-consensus/src/Ouroboros/Consensus/Storage/ChainDB/Impl/ChainSel.hs

```diff
@@ -745,17 +743,28 @@ chainSelectionForBlock cdb@CDB{..} blockCache hdr punish = do
               -- Clear the tentative header
               prevTentativeHeader <- swapTVar varTentativeHeader SNothing
 
-              -- Update the followers
-              --
-              -- 'Follower.switchFork' needs to know the intersection point
-              -- (@ipoint@) between the old and the current chain.
-              let ipoint = castPoint $ Diff.getAnchorPoint chainDiff
-              followerHandles <- Map.elems <$> readTVar cdbFollowers
-              forM_ followerHandles $ \followerHandle ->
-                fhSwitchFork followerHandle ipoint newChain
+              case chainSwitchType of
+                -- When adding a block, the intersection point of the old and
```

Comment:
We might be adding more than one block. The logic is correct still, but the wording shouldn't say "adding a block"